kolibri/core/auth/management/utils.py

```diff
@@ -55,3 +69,96 @@ def get_facility(facility_id=None, noninteractive=False):
                 facility = _interactive_facility_selection()
 
     return facility
+
+
+def get_dataset_id(baseurl, identifier=None):
+    # get list of facilities and if more than 1, display all choices to user
+    facility_url = urljoin(baseurl, reverse("kolibri:core:publicfacility-list"))
+    response = requests.get(facility_url)
+    response.raise_for_status()
+    facilities = response.json()
+    # if provided, look up identifier in list of dataset and facility ids
+    if identifier:
+        for obj in facilities:
+            if identifier == obj["dataset"] or identifier == obj.get("id"):
```

Comment:
Older versions of kolibri do not have the `id` field in the public facilities endpoint, so for backwards compatibility, we must use `get`.